code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -387,6 +390,55 @@
 /obj/machinery/smartfridge/extract/preloaded
 	initial_contents = list(/obj/item/slime_scanner = 2)
 
+// -------------------------
+// Organ Surgery Smartfridge
+// -------------------------
+/obj/machinery/smartfridge/organ
+	name = "smart organ storage"
+	desc = "A refrigerated storage unit for organ storage."
+	max_n_of_items = 20	//vastly lower to prevent processing too long
+	var/repair_rate = 0
+
+/obj/machinery/smartfridge/organ/accept_check(obj/item/O)
+	if(istype(O, /obj/item/organ) && !istype(O, /obj/item/organ/brain))	//would rather not have a machine to freeze brains from a gameplay perspective, if it is so necessary, freezer chests exist
```

Comment:
This needs to be made clear from a player POV.

Reply:
I'll change the description and slip in another line about not being able to accept brains when tried.

Reply:
I may end up changing this later btw, not sure how necessary it is to deny brains in a smartfridge, since the only thing it seems to mitigate is the number of people who'd forget brains of can-be-cloned people in them. Kinda want to see how the initial reaction goes.